gitlab/setup-full.sh

```diff
@@ -1,12 +1,22 @@
 #!/usr/bin/env bash
 IFS=$'\n\t'
-set -euo pipefail
+set -exuo pipefail
+
+##
+# Call this script from Gitlab CI config to bring up all containers,
+# and deploy, resulting in a full functional up-to-date site. It's
+# the slowest setup, but you can run behat and other complex tests.
+#
 
 docker network prune -f && docker network create amazeeio-network
 docker login -u gitlab-ci-token -p $CI_JOB_TOKEN $CI_REGISTRY
 
+docker-compose up -d mariadb
 docker-compose up -d
-docker-compose exec -T test dockerize -wait tcp://mariadb:3306 -timeout 1m
```

Comment:
Turns out this is not needed, and causes bugs in my setup. I am guessing it throws an exit code because mariadb is already running. We don't need to wait if we just start mariadb first, there is PLENTY of time for it to come up.

Reply:
nice!